getUsers.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
```

Comment:
is this file still required at all?

Reply:
Unsure, I will need to discuss with Stanley

Reply:
yeah it is required, we still need the mongoose to make connection to the database

Reply:
sorry, i meant the whole lamdba, not just this line (does the front end still get all the users at any point)

Reply:
File Has been removed for now